ihatemoney/tests/budget_test.py

```diff
@@ -1685,6 +1685,98 @@ def test_session_projects_migration_to_list(self):
             self.assertIsInstance(session["projects"], dict)
             self.assertIn("raclette", session["projects"])
 
+    def test_rss_feed(self):
+        self.post_project("raclette")
+        self.login("raclette")
+
+        self.client.post("/raclette/members/add", data={"name": "george"})
+        self.client.post("/raclette/members/add", data={"name": "peter"})
+        self.client.post("/raclette/members/add", data={"name": "steven"})
+
+        self.client.post(
+            "/raclette/add",
+            data={
+                "date": "2016-12-31",
+                "what": "fromage à raclette",
+                "payer": 1,
+                "payed_for": [1, 2, 3],
+                "amount": "12",
+                "original_currency": "EUR",
+            },
+        )
+        self.client.post(
+            "/raclette/add",
+            data={
+                "date": "2016-12-30",
+                "what": "charcuterie",
+                "payer": 2,
+                "payed_for": [1, 2],
+                "amount": "15",
+                "original_currency": "EUR",
+            },
+        )
+        self.client.post(
+            "/raclette/add",
+            data={
+                "date": "2016-12-29",
+                "what": "vin blanc",
+                "payer": 2,
+                "payed_for": [1, 2],
+                "amount": "10",
+                "original_currency": "EUR",
+            },
+        )
+
+        project = self.get_project("raclette")
+        token = project.generate_token("feed")
+        resp = self.client.get(f"/raclette/feed/{token}.xml")
+        bills = models.Bill.query.all()
+        expected_rss_content = f"""<?xml version="1.0" encoding="utf-8"?>
+<rss version="2.0"
+    xmlns:dc="http://purl.org/dc/elements/1.1/"
+    xmlns:atom="http://www.w3.org/2005/Atom"
+    >
+    <channel>
+        <title>raclette</title>
+        <description>A simple shared budget manager web application</description>
```

Comment:
Well, actually it feels odd to me. You need to state that it's a feed with the latest bills from this project, maybe?